### PR TITLE
bugfix: incorrectly assume the variable not found if the real value is '-'

### PR DESCRIPTION
**Pull request by 3062**
### Issues associated with this PR

#2134

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases. And you need to show the UT result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result.

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: pkg/proxy/var.go [hunk 1]
```diff
@@ -178,7 +178,7 @@ func upstreamLocalAddressGetter(ctx context.Context, value *variable.IndexedValu
 		return info.UpstreamLocalAddress(), nil
 	}
 
-	return variable.ValueNotFound, nil
+	return variable.ValueNotFound, errors.New("not found upstream local address")
```
Comment (taoyuanyuan) [inline pkg/proxy/var.go:181]: 这个 error应该是variable包里的一个const变量，跟variable.ValueNotFound类似

**Comment by 3062** [comment]
@doujiang24 @nejisama please review

**Review by doujiang24 — APPROVED** [review]
+1

**Comment by 3062** [comment]
@taoyuanyuan  please review

**Review by taoyuanyuan — APPROVED** [review]

**Pull request merged**